website/content/docs/concepts/domain-model/targets.mdx

```diff
@@ -81,9 +95,16 @@ TCP targets have the following additional attribute:
 ### SSH target attributes <sup>HCP/ENT</sup>
 
 SSH targets can source username/password or SSH private key credentials from Vault [credential libraries][] or static
-[credentials][]. Boundary then injects credentials into the SSH session between a client and end host. This allows users to
+[credentials][]. If you use injected credentials, Boundary injects credentials into the SSH session between a client and end host. Injected credentials allow users to
```

Comment:
Both the new wording and the previous wording seems a little confusing to me. We might want to make it clear again here that injected application credentials are required for an SSH target. Perhaps something like:

```
SSH targets use injected application credentials to authenticate
an SSH session between the client and end host.
This allows users to securely connect to remote hosts using SSH,
while never being in the possession of a valid credential for that target host.
The injected credentials can be
a username/password or username/private key credential
from Vault [credential libraries][] or static [credentials][],
or a SSH certificate from Vault [SSH certificate credential libraries][].
```